includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -79,14 +97,25 @@ public function register() {
 			function() {
 				$this->set_up_advanced_tracking();
 			},
-			11
+			15
 		);
 		add_filter(
 			'googlesitekit_amp_gtag_opt',
 			function( $gtag_amp_opt ) {
 				return $this->set_up_advanced_tracking_amp( $gtag_amp_opt );
 			}
 		);
+		add_action(
+			'wp_footer',
+			function() {
+				if ( null === $this->measurement_code_injector ) {
+					return;
+				}
```

Comment:
Instead of checking this, let's instantiate the class in here.

Reply:
This issue has been resolved.